packages/web-react/package.json

```diff
@@ -12,9 +12,9 @@
     "url": "https://github.com/lmc-eu/spirit-design-system.git",
     "directory": "packages/web-react"
   },
-  "module": "_es2015/index.js",
```

Comment:
Package structure is treated in a same way as a directory so to use these directories we must also include them in out import path or create and alias.